src/helpers/FormioLoader/FormioLoader.js

```diff
@@ -15,6 +15,8 @@ const requestPluginHandler = (requestArgs, opts) => {
       requestArgs.opts.base = requestArgs.formio.options.base;
       requestArgs.opts.project = requestArgs.formio.options.project;
       requestArgs.opts.namespace = requestArgs.formio.options.namespace;
+      Formio.setProjectUrl(requestArgs.formio.options.project);
```

Comment:
hmm global, that is strange. how many of these 'gotchas' have we come accross.

Reply:
this actual did fix in few places, as in formio.js, there are few ocassions using `Formio.getProjectUrl` to contruct URL for internal request, such as PDF, nested Form, Select (using data resource). (not every component using this method to construct URL tho, eg. ReCaptcha)

by adding `Formio.setProjectUrl` in pre-request hook, hence update the global Formio just before the request, luckily we still can support multiple forms with this patch.

Reply:
Thanks for doing the discovery, could you raise  and issue on the formio js upstream project with examples where they missed local variable instead of global with the proviso to doing multi form on one page with different 'stages/projects'